Cargo.toml

```diff
@@ -3,15 +3,17 @@ name = "staking"
 version = "0.1.1"
 edition = "2021"
 license = "MIT"
+readme = "README.md"
 authors = ["Gear Technologies"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "d4552434" }
```

Comment:
Need to keep `rev = "d4552434"` as we are binding to the concrete `gstd` "release".